Training/view/frontend/templates/account/dashboard/credithold.phtml

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+/**
+ * @var Codifi\Training\Block\Account\Dashboard\CreditHold $block
+ */
+?>
+<?php if ($block->checkForOneTimeDemoMessage()): ?>
+    <div class="confirmation-modal-content">
+        <p><?= $block->escapeHtml($block->getMessage()) ?></p>
+    </div>
+    <script>
+        require([
+            'jquery',
+            'Magento_Ui/js/modal/confirm'
+        ], function ($) {
+            'use strict';
+
+            $('.confirmation-modal-content').confirm({
+                actions: {
+                    always: function() {
+                        $.ajax({
+                            type: "POST",
+                            url: '<?= $block->getSaveUrl() ?>',
+                            data: {
+                                'note': 'Customer is notified about credit hold.',
```

Comment:
All strings should be translated
https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/translations/translate_theory.html#add_strings_js